Removed power 3v3 options

MuddledBox opened the pull request:
# What's new

- Per skotopes, "3.3V: this line is used to power SD-Card it is highly unrecommended to turn this line off(power will be re-enabled to scan sdcard). We will remove this command in future releases." - #1020 

So I removed it here. No longer an option and should not be possible to switch on and off 3v3 via the power command. 

# Verification 

- Tested via PuTTY. Option no longer available and multi-meter does not report voltage changes when command is executed (as command is no longer valid). 

![puttyTest](https://user-images.githubusercontent.com/101580720/158266506-3d24b2c4-0f31-41c5-a24a-379c62c85305.PNG)

# Checklist (do not modify)

- [x] PR has description of feature/bug or link to Confluence/Jira task
- [x] Description contains actions to verify feature/bugfix
- [x] I've built this code, uploaded it to the device and verified feature/bugfix

MuddledBox committed:
Removed power 3v3 options

Per skotopes, "3.3V: this line is used to power SD-Card it is highly unrecommended to turn this line off(power will be re-enabled to scan sdcard). We will remove this command in future releases."

So I removed it here. No longer an option and should not be possible.

DrZlo13 commented:
We need to consult with QA, there is a chance they used this command for production testing.

DrZlo13 commented:
> We need to consult with QA, there is a chance they used this command for production testing.

OK, they dont.

skotopes commented:
Thanks for PR.

One small thing: we have "system debug" flag that we use when do factory tests.
I think it's ok to hide 3.3 control behind this flag so it will be still available for factory testing.

MuddledBox commented:
That sounds like a more elegant solution! Where is the flag and example of its usage?

skotopes commented:
https://github.com/flipperdevices/flipperzero-firmware/blob/3b8b2e59df8bdaac9956ae7521a91ca1f39da705/applications/subghz/subghz_cli.c#L598

MuddledBox committed:
Added Debug Flag Detect

When debug is enabled, "power 3v3" can be used and is shown in the help for "power" menu. When debug is disabled, will not show up and cannot be executed!

MuddledBox commented:
@skotopes Most recent commit now implemented the way you requested. Tested and working via PuTTY, when debug on working as expected, when debug off working as expected. Help menu changes too so no confusion from non-debug users.

skotopes commented:
Please run `make format` to reformat sources

MuddledBox committed:
Update power_cli.c

skotopes committed:
Merge branch 'dev' into remove_power_3v3_option
